apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/TopPanel/TopPanel.tsx

```diff
@@ -13,12 +13,12 @@ import { translations } from '../../../../../locales/i18n';
 const pageTranslations = translations.aaveReserveOverviewPage.topPanel;
 
 type TopPanelProps = {
+  asset: string;
   className?: string;
 };
 
-export const TopPanel: FC<TopPanelProps> = ({ className }) => {
+export const TopPanel: FC<TopPanelProps> = ({ asset, className }) => {
   // TODO: Mocked data
-  const asset = 'BTC';
   const assetName = 'Bitcoin';
```

Comment:
should assetName be part of the asset?

Reply:
Mmm yess, without mocks I think we would have a whole token metadata object. Let me see if I can indicate that here